lib/src/Base/Stat/SobolSequenceDirections.hxx

```diff
@@ -22,10 +22,10 @@
  * which is covered by the LGPL license
  */
 
-const UnsignedInteger    SobolSequence::MaximumNumberOfDimension = 1111;
+const UnsignedInteger    SobolSequence::MaximumNumberOfDimensions = 1111;
```

Comment:
I think we have maximumDimension or maximumIteration more often than maximumNumberOfXXX, maybe use MaximumDimension ?

Reply:
Good idea. Done.